src/YoutubeAtomOverlay.tsx

```diff
@@ -135,7 +135,7 @@ export const YoutubeAtomOverlay = ({
                 >
                     <SvgPlay />
                 </div>
-                {duration && (
+                {duration !== undefined && (
```

Comment:
I think we would need to make sure that it’s a valid length.

```suggestion
                {duration > 0 && (
```

Reply:
I'm not sure which scenario it would be less than 0 but I guess this just makes sure